### PR TITLE
Python 3 support for package

### PR DESCRIPTION
**Pull request by mastizada**
Tested using `python3 setup.py install` and also for python2, both successfully installed

### CODE REVIEW

**Comment by zbraniecki** [comment]
This looks good to me. r+

**Comment by mathjazz** [comment]
@zbraniecki Thanks!

@mastizada I'm thinking about keeping this PR open for now and merging it when we're ready to move Pontoon to Python 3. That way we can keep the tip of the master branch in sync with the revision used in Ponoon, in case we'll need to land any fixes to Silme in the meantime (not that it's likely, but still). Thoughts?

File: ez_setup.py [hunk 1]
```diff
@@ -62,25 +63,27 @@
...
 
 def _validate_md5(egg_name, data):
     if egg_name in md5_data:
         digest = md5(data).hexdigest()
         if digest != md5_data[egg_name]:
-            print >>sys.stderr, (
-                "md5 validation of %s failed!  (Possible download problem?)"
-                % egg_name
+            print(
+                "1md5 validation of %s failed!  (Possible download problem?)"
```
Comment (jotes) [inline ez_setup.py]: `s/1md5/md5/g`

Comment (mastizada) [reply ez_setup.py]: Hi @jotes , what do you mean? To change string to this one?

Comment (mathjazz) [reply ez_setup.py]: @mastizada `1md5` looks like a typo, change it to `md5`.

Comment (jotes) [reply ez_setup.py]: @mathjazz exactly :D  
@mastizada thanks! I could be a little too vague - sorry about that.

Comment (mastizada) [reply ez_setup.py]: good catch actually, I was too sleepy to miss the basic sed reference :)

**Comment by mathjazz** [comment]
Closing on behalf of #2.

**Pull request closed without merging**